package.json

```diff
@@ -38,15 +38,16 @@
   "homepage": "https://github.com/appium/WebDriverAgent#readme",
   "devDependencies": {
     "appium-gulp-plugins": "^4.1.0",
+    "appium-support": "^2.29.0",
```

Comment:
This is already in `dependencies`. And is needed there. Same with `asyncbox`.

Reply:
Fixed it.